.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,48 @@
+# Whether or not rebase-merging is enabled on this repository.
+# Defaults to `true`
+rebaseMergeAllowed: true
+
+# Whether or not squash-merging is enabled on this repository.
+# Defaults to `true`
+squashMergeAllowed: true
+
+# Whether or not PRs are merged with a merge commit on this repository.
+# Defaults to `false`
+mergeCommitAllowed: false
+
+# Rules for master branch protection
+branchProtectionRules:
+# Identifies the protection rule pattern. Name of the branch to be protected.
+# Defaults to `master`
+- pattern: master
+  # Can admins overwrite branch protection.
+  # Defaults to `true`
+  isAdminEnforced: true
+  # Number of approving reviews required to update matching branches.
+  # Defaults to `1`
+  requiredApprovingReviewCount: 1
+  # Are reviews from code owners required to update matching branches.
+  # Defaults to `false`
+  requiresCodeOwnerReviews: true
+  # Require up to date branches
+  requiresStrictStatusChecks: false
```

Comment:
Because CI is both long (~25 minutes) and resource intensive, we have had this flag disabled for this repo.  @kolea2, @busunkim96 please chime in.

Reply:
I think this PR is just making whatever settings we have permanent via the yaml. It looks like this is already set to `false`?

Reply:
Yup

Reply:
`Kokoro` and `cla/google` are the only ones currently required.  If the PR is just supposed to document / specify current policy, I don't think we should be marking the samples / docs-presubmit jobs below required.

Reply:
Sure, updated